ui/decredmaterial/modal.go

```diff
@@ -49,6 +59,28 @@ func (t *Theme) Modal() *Modal {
 	return m
 }
 
+// Dismiss removes the modal from the window. Does nothing if the modal was
+// not previously pushed into a window.
+func (m *Modal) Dismiss() {
+	// ParentWindow will only be accessible if this modal has been
+	// pushed into display by a WindowNavigator.
+	if parentWindow := m.ParentWindow(); parentWindow != nil {
+		parentWindow.DismissModal(m.ID())
+	} else {
+		panic("can't dismiss a modal that has not been displayed")
```

Comment:
i have concerns with the `panic` used here.

Reply:
This would help to catch wrong uses of `modal.Dismiss()` instead of just failing quietly.